lib/server.js

```diff
@@ -106,6 +106,14 @@ module.exports = class Server extends EventEmitter {
         long: 'port'
       },
       {
+        short: 'ip',
```

Comment:
The short version could cause a bit of confusion... maybe use `insPort`?

Reply:
sure